t/export.t

```diff
@@ -0,0 +1,221 @@
+#!/usr/bin/perl -w
+
+# This test scripts:
+# 1. import some products from a CSV file
+# 2. exports the products with various options, and checks that we get the expected exports
+
+use Modern::Perl '2017';
+use utf8;
+
+use Test::More;
+use Log::Any::Adapter 'TAP', filter => "info";
+
+use ProductOpener::Products qw/:all/;
+use ProductOpener::Tags qw/:all/;
+use ProductOpener::PackagerCodes qw/:all/;
+use ProductOpener::Import qw/:all/;
+use ProductOpener::Export qw/:all/;
+use ProductOpener::Config qw/:all/;
+use ProductOpener::Packaging qw/:all/;
+use ProductOpener::Ecoscore qw/:all/;
+use ProductOpener::ForestFootprint qw/:all/;
+use ProductOpener::Data qw/execute_query get_products_collection/;
+
+use File::Path qw/make_path remove_tree/;
+use File::Basename "dirname";
+
+use Getopt::Long;
+use JSON;
+
+
+sub remove_all_products () {
+    # check we are not on a prod database, by checking there are not more than 100 products
+    my $products_count = execute_query(sub {
+		return get_products_collection()->count_documents({});
+	});
+    unless ((0 <= $products_count) && ($products_count < 1000)) {
+        die("Refusing to run destructive test on a DB of more than 100 items");
+    }
+    # clean database
+    execute_query(sub {
+		return get_products_collection()->delete_many({});
+	});
+    # clean files
+    remove_tree("$data_root/products", {keep_root => 1, error => \my $err});
+    if (@$err) {
+        die("not able to remove some products directories: ". join(":", @$err));
+    }
+}
```

Comment:
This should go it a test util, but I didn't know where to put it ! (in a new lib directory ?)